### PR TITLE
A bunch of small fixes

### LINKED ISSUE #6: Use blog post hero photo as meta image

### PR DESCRIPTION
**Pull request by TheOnlyTails**
- [x] Meta images now load correctly based on the page #6 
- [x] The error window now has a min width and height
- [x] Fixed the text wrapping issue on the file preview showcase #14
- [x] Fixed a typo on the file preview showcase
- [x] Use the new "mini" mode of the new Microsoft store popup
- [x] Only use logical CSS properties (change based on writing direction)
- [x] Fix Windows OS detection. #18
- [x] Adapt to the new zip release format on GitHub.
- [x] Fixed inline code wrapping in markdown. #44
- [x] Updated packages.

Closes #6, #14, #18, #44

### CODE REVIEW

File: src/data/features.ts [hunk 1]
```diff
@@ -76,12 +76,12 @@ export const cloudFiles = [
...
 	{
-		name: "June 2018",
+		name: "Files v3",
 		icon: "/ui/icons/folder.png",
-		items: 3,
-		added: "Thursday, June 21, 2018 11:24 AM",
-		modified: "Sunday, November 15, 2020 6:50 PM",
-		path: "C:\\Users\\Sara\\Desktop\\June 2018"
+		items: 16,
+		added: "Wednesday, October 6, 2021 42:69 AM",
```
Comment (chingucoding) [inline src/data/features.ts:82]: While I see that this is quite a funny date, maybe we should stick with a date that would actually work?

File: src/routes/__layout.svelte [hunk 2]
```diff
@@ -112,19 +114,16 @@
...
+	<script>
+		window["clarity"] = window["clarity"] || (() => {
+			(window["clarity"].q = window["clarity"].q || []).push(arguments);
+		});
+
+		let t = document.createElement("script");
+		t.async = true;
+		t.src = "https://www.clarity.ms/tag/4q1wajdktz";
+		let y = document.getElementsByTagName("script")[0];
+		y.parentNode.insertBefore(t, y);
```
Comment (chingucoding) [inline src/routes/__layout.svelte:126]: I assume, this is equivalent to the previous one?

Comment (TheOnlyTails) [reply src/routes/__layout.svelte:126]: It is, I just improved it.

File: src/data/features.ts (hunk 1 above)
Comment (TheOnlyTails) [reply src/data/features.ts:82]: Or, and hear me out: _what if we don't?_

Comment (chingucoding) [reply src/data/features.ts:82]: Than we have a sample that is incorrect and can potentially be seen as offensive or incorrect. But thats only my opinion on this, maybe other folks have other opinions here.

Comment (TheOnlyTails) [reply src/data/features.ts:82]: I don't know, I think 42 and 69 are pretty much considered "meme numbers". If anyone has a problem with it though, I'll definitely change it.

**Comment by TheOnlyTails** [comment]
@chingucoding and I have decided to merge this PR in its current state. All the undone changes will be done in separate PRs.

**Review by chingucoding — APPROVED** [review]
It seems like that the contributor cards are behaving a bit strangely now:

Screenshot of missing contributor cards in the contributors section

Otherwise, the changes look good to me.

**Comment by TheOnlyTails** [comment]
Hmm, I'll take a look. Probably something to do with the scrolling animation direction. Should be an easy fix.

**Comment by TheOnlyTails** [comment]
Should be done now! I've also added the ability to select the tabs in the tab showcase, and a pre-commit Git hook with Husky to automatically upgrade packages before commiting.

**Review by chingucoding — APPROVED** [review]
Looks good to me, thank you for fixing all those bugs!

**Review by chingucoding — APPROVED** [review]

**Review by chingucoding — APPROVED** [review]

**Review by Tropix126 — APPROVED** [review]
LGTM

**Comment by onein528** [comment]
LGTM

**Pull request merged**